### DISCUSSION TITLE
Add input Price and Output price for gemini-1.5-pro and gemini-1.5-flash in Models

### DISCUSSION THREAD

**Discussion by fjeanchar** [langfuse/langfuse#2600] [Support]
When will you add gemini-1.5-pro and gemini-1.5-flash in Models ?

**Comment by davivc** [comment]
Hi, I'm having the same issue. I've tried to add a custom model definition but this didn't work out for me.

![image](https://github.com/user-attachments/assets/dacb506f-3b79-4434-966d-6a1f81de087a)

The current price for Gemini Models can be found here: https://cloud.google.com/vertex-ai/generative-ai/pricing

**Gemini 1.5 Pro**
Input: $0.00125 / 1k characters (when using <= 128k character)
Output: $0.00375 / 1k characters (when using <= 128k character)

**Gemini 1.5 Flash**
Input: $0.000125 / 1k characters (when using <= 128k character)
Output: $0.000375 / 1k characters (when using <= 128k character)

**Comment by booma-stephan** [comment]
Also added custom model pricing for Gemini, but I think the "Maintained by Langfuse" version is prioritized and hence pricing does not work?

**Reply by marcklingen** [reply]
Can you specify? Generally if you add a model definition with the same name and it matches to the same model on a generation via the regex pattern, it will be prioritized over the default model definitions.
